src/components/ProfileParts/ProfileLists.jsx

```diff
@@ -34,54 +34,47 @@ const profileSections = [
     icon: faBell,
   },
   {
-    name: "close account",
+    name: "close-account",
     icon: faBan,
   },
 ];
+
 const ProfileLists = () => {
   const fileInput = useRef(null);
-  const [userPhoto, setUserPhoto] = useState(userInfo.photo); // initial image (Unknown user)
+  const [userPhoto, setUserPhoto] = useState(userInfo.photo);
 
-  // handle the icon click to open the file input
```

Comment:
All these comments you have removed are important for the later maintenance process, so take care the next time to write comments ASAP.